Tutorial01/tutorial01.rb

```diff
@@ -0,0 +1,12 @@
+for a in 1..6
```

Comment:
-Use constant variable for count value ( 5, 6) 
COUNT = 6
for a in 1..COUNT  continue.
for b in 1..COUNT-1